src/www/services_dhcpv6_edit.php

```diff
@@ -207,7 +207,6 @@
                       <input name="hostname" type="text" value="<?=$pconfig['hostname'];?>" />
                       <div class="hidden" data-for="help_for_hostname">
                         <?=gettext("Name of the host, without domain part.");?>
-                        <?=gettext("If no IP address is given above, hostname will not be visible to DNS services with lease registration enabled.");?>
```

Comment:
@gwjo @AdSchellevis might not work for dhcpv6 just yet. we don't have parity between newer unbound dhcpv4 reader and the legacy readers dhcpleases and dhcpleases6. Looking closely at dhcpleases6 it looks like it doesn't do anything at all?

https://github.com/opnsense/ports/blob/master/opnsense/dhcpleases/files/dhcpleases6.c

Reply:
Ok it does one thing only...run a given command https://github.com/opnsense/ports/blob/d92b0a3845d4a6441054fd25012e28eb1a11a8b0/opnsense/dhcpleases/files/dhcpleases6.c#L559